language/stdlib/modules/DiemSystem.move

```diff
@@ -75,10 +75,15 @@ module DiemSystem {
     const EVALIDATOR_INDEX: u64 = 5;
     /// Rate limited when trying to update config
     const ECONFIG_UPDATE_RATE_LIMITED: u64 = 6;
+    /// There was an issue in the addition/removal of a validator to/from the validator set
+    const EVALIDATOR_SET: u64 = 7;
```

Comment:
`EMAX_VALIDATORS` might be a bit clearer? "limit exceeded: max validators" clearly conveys what's happening, but "limited exceeded: validator set" sounds like it could potentially be too few validators or some other problem.

Reply:
Good suggestion! I'll make the change